src/components/Table/utils/TableState.ts

```diff
@@ -90,6 +101,61 @@ export class RowState {
     this.selectedRows.merge(map);
   }
 
+  initSortState(sortConfig: GridSortConfig | undefined, columns: GridColumnWithId<any>[]) {
+    if (!this.sortConfig) {
```

Comment:
Thought: Really minor but `if (this.sortConfig) { return }` would let the rest of this method de-dent by one.